letsencrypt/default_server.nginx.jinja

```diff
@@ -0,0 +1,32 @@
+# default servers for nginx
+# this vHosts will catch all request that are not match an other vHosts server_name.
+server {
+    listen *:80    default_server;
+    listen [::]:80 default_server;
+    return 301 https://{{ fqdn }}$request_uri;
+}
+
+server {
+    listen *:443 default_server;
+    listen [::]:443 default_server;
+
+    ssl_protocols TLSv1.3;
```

Comment:
I think this section is the most hard to get right. While all of those settings are quite solid defaults, I'm not sure if in this case we should revert to `nginx` defaults, as we do not know what the other sides are configured like?

Might also be just OK to leave these (even though TLSv1.3 is not supported by all common clients as of today), or make it configurable in a pillar if need be (with reasonable defaults)?

Not entirely sure how to do this nicely. Ideas?

Reply:
I removed the TLS setting, because I think it makes sense to fail back to the nginx default.

Reply:
Yes. We can create a global setting for ciphers in a seperate pull request.